### ISSUE TITLE
use nosetests instead of unittest

### ISSUE THREAD

**Issue by halfak** [tapilab/json2tsv#11]
I propose that we switch from unittest to nosetests.  Nosetests can still run unittest tests, but it also enables much more flexible testing runs and more simplistic syntax.

**Comment by aronwc** [comment]
Yes, I like nose; also can run doctests, which I like (for simple testing).

**Comment by halfak** [comment]
COol.  I'll work it out. :)
